classes/PodsAdmin.php

```diff
@@ -3741,6 +3741,10 @@ public function add_debug_information( $info ) {
 					'label' => __( 'Pods API Cache Activated' ),
 					'value' => ( pods_api_cache() ) ? __( 'Yes', 'pods' ) : __( 'No', 'pods' ),
 				),
+				'pods-shortcode-allow-evaluate-tags'                  => array(
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'pods-shortcode-allow-evaluate-tags'" and double arrow, but found 18.

<sub>hash: c4e4274379c3436ca550a7cad8cb6b12</sub>